mmv1/third_party/terraform/services/compute/resource_compute_instance_group_membership_test.go.erb

```diff
@@ -0,0 +1,227 @@
+<% autogen_exception -%>
+package compute_test
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func TestAccComputeInstanceGroupMembership_instanceGroupMembershipBasic(t *testing.T) {
+	// Multiple fine-grained resources
+	acctest.SkipIfVcr(t)
+	t.Parallel()
+
+  suffix := acctest.RandString(t, 10)
```

Comment:
Nit: Indentation in this file could be cleaned up for readability

Reply:
Yeah, Intellij showed me different indentations - fixed